hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/RDBStore.java

```diff
@@ -327,6 +329,44 @@ public CodecRegistry getCodecRegistry() {
     return codecRegistry;
   }
 
+  @Override
+  public DBUpdatesWrapper getUpdatesSince(long sequenceNumber)
+      throws SequenceNumberNotFoundException {
+
+    DBUpdatesWrapper dbUpdatesWrapper = new DBUpdatesWrapper();
+    try {
+      TransactionLogIterator transactionLogIterator =
+          db.getUpdatesSince(sequenceNumber);
+
+      boolean flag = true;
+
+      while (transactionLogIterator.isValid()) {
+        TransactionLogIterator.BatchResult result =
+            transactionLogIterator.getBatch();
+        long currSequenceNumber = result.sequenceNumber();
+        if (flag && currSequenceNumber > 1 + sequenceNumber) {
+          throw new SequenceNumberNotFoundException("Unable to read data from" +
+              " RocksDB wal to get delta updates. It may have already been" +
+              "flushed to SSTs.");
+        }
+        flag = false;
+        if (currSequenceNumber == sequenceNumber) {
+          transactionLogIterator.next();
+          continue;
+        }
+        WriteBatch writeBatch = result.writeBatch();
+        byte[] writeBatchData = writeBatch.data();
```

Comment:
Nitpick: we can avoid temporary variables writeBatch and writeBatchData.

Reply:
done